### ISSUE TITLE
[Requests] Can you provide this as `npm` package?

### ISSUE THREAD

**Issue by biniama** [sean-perkins/nativescript-onboarding#1]
Dear Sean,

Thanks for the wonderful on-boarding project.

I had to create a new project and move files around which created conflict.
Then, I cloned your project as my new project and edited it to fit my need.

For others who want to use your project, can you provide the on-boarding, fonts and others as an NPM module so it is easier and also provide example of using the npm module with the `+home` component.

*Another request:* (I don't want to create multiple issues)
I want to slide the pages using slide bar not using `continue` button. Is that possible to do?

**Comment by sean-perkins** [comment]
Making this into an npm package is possible, basically I would need to exclude out the `OnboardingModule`, but things like fonts, etc. would have to be manually copied in via a post install script or something similar. 

Can you elaborate on the "slide bar"? Is that a native control or the swipe gesture on the screen?

**Comment by biniama** [comment]
Thanks Sean for the kind answer.

By slide bar, I meant the slide gesture like it is found in most apps with
the possibility to go to the previous slide as well.

Regards

**Comment by sean-perkins** [comment]
Yes, that should be possible. Although may require some fiddling. Ideally you could listen for those pan gestures on the slide element and then re-use the "next" and "previous" function hooks to control the changes.

If I get some availability this weekend, I'll dig into that and apply additional options for it.

**Comment by biniama** [comment]
Dear Sean,

Any update?

On Nov 30, 2017 5:59 PM, "Sean Perkins" <notifications@github.com> wrote:

Yes, that should be possible. Although may require some fiddling. Ideally
you could listen for those pan gestures on the slide element and then
re-use the "next" and "previous" function hooks to control the changes.

If I get some availability this weekend, I'll dig into that and apply
additional options for it.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/sean-perkins/nativescript-onboarding/issues/1#issuecomment-348252043>,
or mute the thread
